requirements/base.in

```diff
@@ -1,8 +1,8 @@
 # Core requirements for using this application
 
-celery>=3.0,<4.0          # Asynchronous tasks
-edx-celeryutils           # Custom task extensions
-Django>=1.11              # Web application framework
+celery==3.1.18            # Asynchronous tasks
+edx-celeryutils<0.2      # Custom task extensions
+Django>=1.8              # Web application framework
```

Comment:
These versions are required to make this work with ginkgo.

Reply:
The requirements/*.txt changes below are the result of running `make upgrade` on the new requirements versions.